src/backend/src/services/teams.services.ts

```diff
@@ -418,6 +427,47 @@ export default class TeamsService {
     return teamTypes;
   }
 
+  /**
+   * Changes the description of the given teamType to be the new description
+   * @param user The user who is editing the description
+   * @param teamTypeId The id for the teamType that is being edited
+   * @param name the new name for the team
+   * @param iconName the new icon name for the team
+   * @param description the new description for the team
+   * @param organizationId The organization the user is currently in
+   * @returns The team with the new description
+   */
+  static async editTeamType(
```

Comment:
Make sure to update the doc comment to include what the filepath is, also you should probably rename it here and wherever else to be imageFilePath or similar